packages/@aws-cdk/alexa-ask/package.json

```diff
@@ -82,11 +82,11 @@
   },
   "dependencies": {
     "@aws-cdk/core": "0.0.0",
-    "constructs": "^3.2.0"
+    "constructs": "10.0.0-pre.5"
```

Comment:
Um. Is this intentional and acceptable? 

Feels dodgy. Won't this force the same prerelease dependency on our customers?

Reply:
Yes, intentional and acceptable. Bear in mind that this is merging into v2 (which is pre-release in itself). Also, it is impossible to take caret dependencies on pre-releases anyway.